### ISSUE TITLE
dialog dependency missing

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#146]
In GitLab by @ rbakhshi on Jun 16, 2016, 10:43

The current version of autosubmit (3.6.1) should require the dialog package (of Debian) since dialog is used for the configuration step. Without this package installed, I get the following error:
```
$ autosubmit configure
[CRITICAL] Unhandled exception on Autosubmit:
Traceback (most recent call last):
   File "/usr/local/lib/python2.7/dist-packages/autosubmit/autosubmit.py", line 306, in parse_args
     return Autosubmit.configure()
   File "/usr/local/lib/python2.7/dist-packages/autosubmit/autosubmit.py", line 1066, in configure
     d = dialog.Dialog(dialog="dialog", autowidgetsize=True, screen_color='GREEN')
   File "/usr/local/lib/python2.7/dist-packages/dialog.py", line 1015, in __init__
     self._dialog_prg = _path_to_executable(dialog)
   File "/usr/local/lib/python2.7/dist-packages/dialog.py", line 479, in _path_to_executable
     "can't find the executable for the dialog-like "
ExecutableNotFound 
```
Note that I have python2-pythondialog package installed, but the problem persists.

**Comment by kinow** [comment]
In GitLab by @charlio86 on Jul 7, 2016, 09:39

I also had this problem, but after install dialog (sudo apt-get install dialog) this problem was solved

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jul 8, 2016, 17:18

In order to avoid this confusion (missing dependency), we will add the dependency on the documentation soon (next release).

Thanks @charlio86 & @ rbakhshi !!!

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jul 22, 2016, 12:19

@lbatte

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jul 25, 2016, 12:58

I think we should re-implement the command line option for the configure step (if dialog is available ok, but if not read dbname, dbpath, rootpath, defaultjobs and defaultplatforms from argparse parameters).

**Comment by kinow** [comment]
In GitLab by @dmanubens on Jul 28, 2016, 12:10

Status changed to closed by commit c5be98dbd70879f7823adcf4876591fd007769fd
